emcc.py

```diff
@@ -2506,10 +2502,6 @@ def phase_linker_setup(options, state, newargs):
       'removeRunDependency',
     ]
 
-  if options.embind_emit_tsd:
-    # TODO: Remove after #19759 is resolved.
-    settings.REQUIRED_EXPORTS += ['free']
```

Comment:
I'm not sure what this embind option is. Where is the dep for it that replaces this?

Reply:
I believe its: `src/embind/embind_shared.js:  $getTypeName__deps: ['$readLatin1String', '__getTypeName', 'free'],`